files/content/bonce/filenames.txt

```diff
@@ -0,0 +1,573 @@
+./acidshooter.xml
```

Comment:
this can easily be `filenames.lua`, just need to replace all `\n` with `",\n"`, end with `}`, start with `return {`. In code, `filenames = dofile("mods/noita.fairmod/files/content/bonce/filenames.lua")`